src/LiveComponent/src/DependencyInjection/Compiler/OptionalDependencyPass.php

```diff
@@ -33,5 +33,10 @@ public function process(ContainerBuilder $container): void
                 ->addTag(LiveComponentBundle::HYDRATION_EXTENSION_TAG)
             ;
         }
+
+        if (!$container->hasDefinition('test.client')) {
+            $container->getDefinition('ux.live_component.event_subscriber')
+                ->setArgument(1, false);
```

Comment:
In test mode, we don't require the custom `Accept` content-type to make testing easier (and not break existing tests)